pkg/operator/resources/operator/operator.go

```diff
@@ -726,6 +726,9 @@ attachment definitions on kubevirt by using vnic profile to network attachment d
 																		"type": {
 																			Type: "string",
 																		},
+																		"volumeMode": {
```

Comment:
The description of the object needs to be fixed; it refers to `ResourceMappingItem` which no longer exist ("ResourceMappingItem defines the...") (line 699).